static/js/src/ejsImageGallery.js

```diff
@@ -102,6 +129,25 @@ class ImageGallery {
             });
     }
 
+    _parseVideoId(url) {
+        let videoId = null
+
+        return new Promise((resolve, _) => {
+            const a = document.createElement('a')
+            a.href = url
+
+            if (a.hostname === 'youtu.be') {
+                videoId = a.pathname.substr(1, a.pathname.length)
```

Comment:
Add a comment within the `if` block providing an example URL matching this condition.